packages/components/src/__actions__/Menu/v3/MenuTrigger.tsx

```diff
@@ -0,0 +1,14 @@
+import React from "react"
+import {
+  MenuTrigger as RACMenuTrigger,
+  MenuTriggerProps as RACMenuTriggerProps,
+} from "react-aria-components"
+
+export type MenuTriggerProps = Omit<RACMenuTriggerProps, "trigger">
+
+/**
+ * A MenuTrigger adds open/close functionality when wrapping a Button and a Popover (with a Menu inside of the Popover)
+ */
+export const MenuTrigger = (props: MenuTriggerProps): JSX.Element => (
```

Comment:
FYI the trigger components from RAC don't accept a `ref` prop, so I haven't done a forwardRef for them